client/asset/dcr/dcr.go

```diff
@@ -2912,6 +2833,19 @@ func (dcr *ExchangeWallet) handleTipChange(newTipHash *chainhash.Hash, newTipHei
 	go dcr.findRedemptionsInBlockRange(startHeight, newTipHeight, contractOutpoints)
 }
 
+func (dcr *ExchangeWallet) getMainchainHeader(blockHash *chainhash.Hash) (*wire.BlockHeader, bool, error) {
```

Comment:
The header returned isn't necessarily a mainchain header. Method name might be confusing.